### ISSUE TITLE
Looks like gradle is defaulting to `build-logic` instead of `gradle/plugins`

### ISSUE THREAD

**Issue by yogurtearl** [jjohannes/idiomatic-gradle#11]
Looks like gradle is defaulting to `build-logic` instead of `gradle/plugins`, maybe this repo should follow suit? 

See https://github.com/gradle/gradle/pull/22141/files

**Comment by jjohannes** [comment]
I am not really sure why this change was made. I think putting things in a `gradle/...` folder is a good advice in general. There was also agreement from members of the Gradle team and I have not seen a good argument against that.

Similar to #8 though, one could think about other names _inside_ `gradle/...`.

I currently still like `gradle/plugins` most of all options I have seen proposed so far.

**Comment by yogurtearl** [comment]
personally, I do prefer `gradle/plugins` over `build-logic` but not sure if it is "idiomatic" if it differs from the gradle docs and from the `gradle init` new project defaults. 

Can cause confusion with devs when it doesn't match the gradle docs.

**Comment by gildor** [comment]
I agree,  that I like gradle/plugins a lot more, I don't need one more dir on root level in addition to my 300 module dirs.

I think it would be great if we started discussion on [gradle/gradkle#](https://github.com/gradle/gradle/issues/14048)
